ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino

```diff
@@ -28,32 +28,33 @@ DNSServer dns;
 #define MAX_SUPPORTED_EXTRUDERS 6       // Number of supported extruder
 #define USE_FAST_SD                     // Use Default fast SD clock, comment if your SD is an old or slow one.
 //#define OTA_UPDATES                   // Enable OTA firmware updates, comment if you don't want it (OTA may lead to security issues because someone may load any code on device)
-const int serialBauds[] = { 1000000, 500000, 250000, 115200, 57600 };   // Marlin valid bauds (removed very low bauds)
+const uint16_t serialBauds[] = { 1000000, 500000, 250000, 115200, 57600 };   // Marlin valid bauds (removed very low bauds)
```

Comment:
just realized int and uint16_t are terrible wrong here! should be uint32_t

Reply:
That's true not sure but on 32 system int is 32bits by default...this is my big fault to use uint16_t